src/review/review.service.ts

```diff
@@ -32,14 +32,14 @@ export class ReviewService {
     const { hotLevel, userId, foodId, tagIds } = reviewDetails;
     const review = new Review();
     const hotLevelId = produceHotLevelId(hotLevel);
-
     review.hotLevel = await getRepository(FoodLevel).findOne(hotLevelId);
     review.user = await getRepository(User).findOne(userId);
     review.food = await getRepository(Food).findOne(foodId);
     review.tasteReviews = await Promise.all(
-      tagIds.map(async (tagid) => {
-        const tag = getRepository(TasteTag).findOne(tagid);
-        return tag;
+      tagIds.map(async (tag) => {
+        const tagid = produceTasteTagId(tag)
+        const Tag = getRepository(TasteTag).findOne(tagid);
```

Comment:
```suggestion
        const tagId = produceTasteTagId(tag)
        const Tag = getRepository(TasteTag).findOne(tagId);
```
`tagid` -> `tagId`로 수정이 필요 합니다!